doc/website/sidebars.js

```diff
@@ -35,6 +35,7 @@ module.exports = {
         'predicates',
         'aggregate',
         'paging',
+        'extension-api'
```

Comment:
The current website structure uses the "Code Generation" section for documenting the generated API. I don't like it, but I prefer the new extension doc will reside close to the "Templates" page (on "Misc").

We already agreed the website needs a new structure, so we can change it in the future.

Reply:
kk